app/events/admin.go

```diff
@@ -479,6 +480,31 @@ func (a *admin) callbackUnbanConfirmed(query *tbapi.CallbackQuery) error {
 	if err := send(editMsg, a.tbAPI); err != nil {
 		return fmt.Errorf("failed to edit message, chatID:%d, msgID:%d, %w", chatID, query.Message.MessageID, err)
 	}
+
+	if a.restoreMsg != "" {
+		// send the original message text back to the main chat and meniton the original poster
+		fmtName := ""
+		if name != "" {
+			fmtName = fmt.Sprintf("[%s](tg://user?id=%d)", name, userID)
+		} else {
```

Comment:
the logic with nested if/else is hard to read and follow. Can you simplify it by making it a single-level and preferable without else.